File: ExercicioMetodos/src/Calculadora.java
public class Calculadora {
    public static void soma (double num1, double num2) {
        double resultado = num1 + num2;
        System.out.println("A soma de " + num1 + " + " + num2 + " = " + resultado);
    }

    public static void subtracao (double num1, double num2) {
        double resultado = num1 - num2;
        System.out.println("A diferença de " + num1 + " - " + num2 + " = " + resultado);
    }

    public static void multiplicacao (double num1, double num2) {
        double resultado = num1 * num2;
        System.out.println("O produto de " + num1 + " x " + num2 + " = " + resultado);
    }

    public static void divisao (double num1, double num2) {
        double resultado = num1 / num2;
        System.out.println("O quociente de " + num1 + " / " + num2 + " = " + resultado);
    }
}

File: ExerciciosMetodos/Calculadora.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExerciciosMetodos
{
    public class Calculadora
    {
        public static void Soma( double num1, double num2)
        {
            double resultado = num1 + num2;
            Console.WriteLine("A soma de {0} + {1} = {2}", num1, num2, resultado);
        }

        public static void Subtracao(double num1, double num2)
        {
            double resultado = num1 - num2;
            Console.WriteLine("A diferença entre {0} - {1} = {2}", num1, num2, resultado);
        }

        public static void Multiplicacao(double num1, double num2)
        {
            double resultado = num1 * num2;
            Console.WriteLine("O Produto de {0} x {1} = {2}", num1, num2, resultado);
        }

        public static void Divisao(double num1, double num2)
        {
            double resultado = num1 + num2;
            Console.WriteLine("O quociente de {0} / {1} = {2}", num1, num2, resultado);
        }
    }
}
